src/Controls/src/SourceGen/RouteSourceGenerator.cs

```diff
@@ -0,0 +1,254 @@
+﻿using System.Linq;
+using System.Text;
+
+using Microsoft.CodeAnalysis;
+
+using Microsoft.Maui.Controls;
+
+using static Microsoft.Maui.Controls.SourceGen.Helper;
+
+namespace VijayAnand.MauiAutomation
```

Comment:
Wrong namespace? 😁

Reply:
Also in some other files.

Reply:
My bad. Sorry for that. Will push another commit to correct the wrong namespace.